plugins/MultiSites/API.php

```diff
@@ -343,9 +343,9 @@ private function buildDataTable($sitesToProblablyAdd, $period, $date, $segment,
                 'ColumnCallbackDeleteRow',
                 array(
                      self::NB_VISITS_METRIC,
-                     function ($value) {
-                         return $value == 0;
-                     }
+                        function ($value) {
```

Comment:
Same as above here, this shouldn't be indented but because `self::NB_VISITS_METRIC` is 5 spaces then this is one level further unnecessarily.